cmd/chainbridge/main.go

```diff
@@ -41,25 +36,37 @@ var devFlags = []cli.Flag{
 }
 
 var accountCommand = cli.Command{
-	Action:   handleAccountsCmd,
 	Name:     "accounts",
 	Usage:    "manage bridge keystore",
-	Flags:    accountFlags,
 	Category: "KEYSTORE",
 	Description: "The account command is used to manage the bridge keystore.\n" +
-		"\tTo generate a new secp256k1 (Ethereum) account: bridge account --generate\n" +
-		"\tTo import a keystore file: bridge account --import=path/to/file\n" +
-		"\tTo list keys: bridge account --list",
+		"\tTo generate a new secp256k1 (Ethereum) account: bridge account generate\n" +
+		"\tTo import a keystore file: bridge account import path/to/file\n" +
+		"\tTo list keys: bridge account list",
```

Comment:
Please can we include that `generate` has other key types, right now it's not very obvious when looking at `chainbridge accounts --help`:
```
d@P51:~/dev/ChainBridgeV2$ chainbridge accounts --help
NAME:
   chainbridge accounts - The account command is used to manage the bridge keystore.
  To generate a new secp256k1 (Ethereum) account: bridge account generate
  To import a keystore file: bridge account import path/to/file
  To list keys: bridge account list

USAGE:
   chainbridge accounts command [command options] [arguments...]

COMMANDS:

   KEYSTORE:
     generate  generate bridge keystore
     import    import bridge keystore
     list      list bridge keystore

OPTIONS:
   --help, -h  show help
```
It should also be chainbridge not bridge.